### PR TITLE
Updated Fast DDS documentation links to latest version.

### PR DESCRIPTION
**Pull request by rsanchez15**

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/7/)

File: docs/examples/partition.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ If a publisher with a partition is deployed, only the subscriber with the same p
 In this demo, there are four partitions (``A``, ``B``, ``C`` and ``D``).
 Additionally, you can select the ``*`` partition, which refers to all partitions.
 Please refer to
-`Fast DDS Partitions Documentation <https://fast-dds.docs.eprosima.com/en/v2.0.0/advanced.html?#partitions>`_
+`Fast DDS Partitions Documentation <https://fast-dds.docs.eprosima.com/en/latest/advanced.html?#partitions>`_
```
Comment (EduPonz) [inline docs/examples/partition.rst:11]:
```suggestion
`Fast DDS Partitions Documentation <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/domain/domainParticipant/partition.html#partitions>`_
```

File: docs/first_steps/first_steps.rst [hunk 2]
```diff
@@ -23,18 +23,18 @@ There are multiple parameters that the user can define in this menu:
...
-  `Fast DDS Topics with key documentation <https://fast-dds.docs.eprosima.com/en/v2.0.0/fastdds/dds_layer/topic/typeSupport/typeSupport.html#data-types-with-a-key>`_).
+  `Fast DDS Topics with key documentation <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/topic/typeSupport/typeSupport.html#data-types-with-a-key>`_).
 
 - **Size:** This parameter allows to control the size of the shape. The size can vary between ``1`` and ``99``.
 
 - **Partition:** The user can select different partitions to differentiate groups of publishers and subscribers.
   The user can select between four partitions (``A``, ``B``, ``C`` and ``D``).
   Additionally the user can select the ``*`` partition, that will be matched against all other partitions (see
-  `Fast DDS Partitions Documentation <https://fast-dds.docs.eprosima.com/en/v2.0.0/advanced.html?#partitions>`_).
+  `Fast DDS Partitions Documentation <https://fast-dds.docs.eprosima.com/en/latest/advanced.html?#partitions>`_).
```
Comment (EduPonz) [inline docs/first_steps/first_steps.rst:37]:
```suggestion
  `Fast DDS Partitions Documentation <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/domain/domainParticipant/partition.html#partitions>`_).
```

File: docs/installation/linux_sources.rst [hunk 3]
```diff
@@ -16,7 +16,7 @@ To install eProsima Shapes Demo using colcon, please follow the steps below:
 
 1.  Install the eProsima Fast DDS dependencies and verify that the system meets the installation requirements.
     The complete list of requirements and dependencies can be found in the
-    `Fast DDS Linux Installation Manual <https://fast-dds.docs.eprosima.com/en/v2.0.0/installation/sources/sources.html>`_.
+    `Fast DDS Linux Installation Manual <https://fast-dds.docs.eprosima.com/en/latest/installation/sources/sources.html>`_.
```
Comment (EduPonz) [inline docs/installation/linux_sources.rst:19]:
```suggestion
    `Fast DDS Linux Installation Manual <https://fast-dds.docs.eprosima.com/en/latest/installation/sources/sources_linux.html>`_.
```

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/9/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/11/)

**Pull request merged**